README.md

```diff
@@ -54,6 +54,18 @@ This is the capstone project for react. It consumes an API, uses Redux as a stat
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+### Video Demo <a name="video-demo"></a>
+
+[Video Link](https://www.loom.com/share/36ccb36709224b2ba879bc5bfc76cb63?sid=d50db147-b8c9-44eb-bf1d-f2bd069e3512)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] very nice demo video, You present the application very clearly, but according to the project requirements, you should also include a highlight about a piece of the code you find interesting or are proud of making.